catalog/app/containers/Bucket/PackageUpdateDialog.tsx

```diff
@@ -275,7 +325,6 @@ function DialogForm({
         error,
         submitError,
         hasValidationErrors,
-        form,
```

Comment:
Hmmm… Eslint didn't catch this, because of `<form`? :thinking:

Reply:
yeah, probably